### PR TITLE
Removes fungus announcement

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
This PR removes the fungus biohazard station announcement. (The event itself remains.)

## Why It's Good For The Game
This announcement is really over the top for the severity of the event. Calling it a "biohazard" is a bit much too considering that is strongly associated with Terrors or Blob.
All it serves to do is cause confusion in newer players or just disrupt the chat log for no reason.

## Changelog
:cl:
del: Removed large station alert regarding fungus.
/:cl:

### CODE REVIEW

**Comment by Wojz** [comment]
> Crewmate victory

Is this because there's a fungus among us.... Sean...

File: code/modules/events/wallrot.dm [hunk 1]
```diff
@@ -1,9 +1,5 @@
 /datum/event/wallrot/setup()
```
Comment (farie82) [inline code/modules/events/wallrot.dm:1]: Can be removed.
The event only has to start. Then it can be killed again. It would be a waste to have it tick while not doing anything.
If both `startWhen` and `endWhen` are 0, which is the default, then it'll only call `start` and then afterwards instantly call `end`

File: code/modules/events/wallrot.dm [hunk 2]
```diff
@@ -1,9 +1,5 @@
...
-	announceWhen = rand(0, 300)
-	endWhen = announceWhen + 1
-
-/datum/event/wallrot/announce()
-	GLOB.event_announcement.Announce("Harmful fungi detected on station. Station structures may be contaminated.", "Biohazard Alert")
+	endWhen = rand(1, 300)
```
Comment (farie82) [inline code/modules/events/wallrot.dm:2]: Or at least this one can be set to `0` instead of a random value

**Review by farie82 — CHANGES_REQUESTED** [review]

File: code/modules/events/wallrot.dm (hunk 1 above)
Comment (S34NW) [reply code/modules/events/wallrot.dm:1]: Mm, I wasn't familiar enough with event code to know if it did things constantly through the lifetime of the event, or if it just did it at the start.

**Review by dearmochi — APPROVED** [review]

**Comment by AffectedArc07** [comment]
image

🗞️

**Review by SteelSlayer — APPROVED** [review]
250px-591Amoonguss

File: code/modules/events/wallrot.dm [hunk 3]
```diff
@@ -1,10 +1,3 @@
-/datum/event/wallrot/setup()
```
Comment (Aurorablade) [inline code/modules/events/wallrot.dm:1]: I would not remove the annoucement i would reword it. https://github.com/ParadiseSS13/Paradise/blob/b151dfb0b9b5b53e41a5a2605d1f38471a8e87df/code/game/turfs/simulated/walls.dm#L112

Wallrot weakens the damage walls can take before you can break it. so yeah it can be an issue.
Rather then biohazard perhaps something about station structural integrity?

Comment (S34NW) [reply code/modules/events/wallrot.dm:1]: Do people actively hunt for wallrot after hearing the announcement? As far as I'm aware, everyone ignores it and will only (sometimes) deal with it if they organically come across it.

Comment (Sirryan2002) [reply code/modules/events/wallrot.dm:1]: > Do people actively hunt for wallrot after hearing the announcement? As far as I'm aware, everyone ignores it and will only (sometimes) deal with it if they organically come across it.

people don't for the most part

Comment (Aurorablade) [reply code/modules/events/wallrot.dm:1]: if chem needs fungus for something. but REGARDLESS i think a rewording would be preferable to an outright removal

Comment (Wojz) [reply code/modules/events/wallrot.dm:1]: > Do people actively hunt for wallrot after hearing the announcement? As far as I'm aware, everyone ignores it and will only (sometimes) deal with it if they organically come across it.

It gets dealt with but never on purpose from what I've seen.

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]
Probably the most useless announcement in the entire game
